bcda/auth/models_test.go

```diff
@@ -63,6 +63,19 @@ func (s *ModelsTestSuite) TestTokenCreation() {
 	assert.Equal(s.T(), tokenString, savedToken.TokenString)
 }
 
+func (s *ModelsTestSuite) TestRevokeSystemKeyPair() {
+	system := models.System{GroupID: "A00001"}
+	s.db.Save(&models.Group{GroupID: "A00001"})
+	s.db.Save(&system)
```

Comment:
On our earlier topic of unit test data, we might want to delete the created System and Group at the end of this test.